include/internal/ktls.h

```diff
@@ -119,6 +205,108 @@ static ossl_inline ossl_ssize_t ktls_sendfile(int s, int fd, off_t off,
     }
     return sbytes;
 }
+
+#   ifdef OSSL_SSL_LOCAL_H
```

Comment:
This guard permits using members of `SSL` in these functions.  This avoids having to pass several members as parameters, and in practice these two functions aren't suited for use outside of libssl unlike the other functions this header defines.